drivers/i3c/i3c_npcx.c

```diff
@@ -0,0 +1,2055 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*132*/
+#define DT_DRV_COMPAT nuvoton_npcx_i3c
+
+#include <string.h>
+
+#include <zephyr/device.h>
+#include <zephyr/irq.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/sys/sys_io.h>
+
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/i3c.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/reset.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(npcx_i3c, CONFIG_I3C_LOG_LEVEL);
+
+#define NPCX_I3C_CHK_TIMEOUT 10000    /* Unit: us, timeout for checking register status */
+#define I3C_SCL_PP_FREQ_MAX  12500000 /* Unit: Mhz*/
+#define I3C_SCL_OD_FREQ_MAX  4170000  /* Unit: Mhz*/
```

Comment:
use the unit in the define naming, drop the comment, `I3C_SCL_OD_FREQ_MAX_MHZ` etc

Reply:
Done